thrift/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -97,6 +102,21 @@
 
     private static final SerializationFormat[] EMPTY_FORMATS = new SerializationFormat[0];
 
+    /**
+     * Creates a new instance of {@link THttpServiceBuilder} which can build an instance of {@link THttpService}
+     * fluently.
+     *
+     * <p>The default SerializationFormat {@link ThriftSerializationFormats#BINARY} will be used when client
```

Comment:
nit: `{@link SerializationFormat}`

Reply:
`{@link SerializationFormat} {@link ThriftSerializationFormats#BINARY BINARY}` ?

Reply:
I'm not sure double link is good. It seems weird. 😅
Let me resolve this.